source/agora/consensus/data/genesis/package.d

```diff
@@ -65,9 +65,9 @@ unittest
 /// Check the Test Genesis Block enrollments (prints replacement enrollments if needed for agora.consensus.data.genesis.Test.d)
 unittest
 {
-    import agora.consensus.data.genesis.Coinnet : GenesisBlock;
+    import agora.consensus.data.genesis.Coinnet : GenesisBlock, coinet_genesis_validator_keys;
 
-    checkGenesisEnrollments(GenesisBlock, genesis_validator_keys, 1008);
+    checkGenesisEnrollments(GenesisBlock, coinet_genesis_validator_keys, 1008);
```

Comment:
There is supposed to be a symmetry between modules in `agora.consensus.data.genesis`.
It seems like `genesis_validator_keys` was breaking that symmetry.
The better fix is to move it from `agora.consensus.data.Block` to `agora.consensus.data.genesis.Test` and to remove the `coinet` prefix in the `Coinnet` module.